src/components/clusters/NewClusterPage.tsx

```diff
@@ -49,6 +51,10 @@ const NewClusterForm: React.FC<NewClusterFormProps> = ({ pullSecret = '', versio
     nameInputRef.current?.focus();
   }, []);
 
+  const [highAvailabilityMode, setHighAvailabilityMode] = React.useState<
```

Comment:
I think the state can be managed as any other Formik's value.

Reply:
it's a bit problematic with Formik since formik's checkbox only have boolean values while in this case we need a string ('Full' or 'None') as a value.
moreover, since the values ([here](https://github.com/openshift-assisted/assisted-ui-lib/pull/421/files/cbbb5598b40f99d92a0f01b856ce3f9736beac49#diff-7ef7e9bedb8f975ebe0cde62f30576ce0adf82924b76e97cc314259a4953a453R69)) are `ClusterCreateParams` `highAvailabiltyMode` cannot be boolean.

Reply:
I agree with Marek that the field should be managed by Formik as any other field. There are 2 options:

1/ Use existing `CheckboxField` component and use it to `setFieldValue` as we do for example in `NetworkConfiguration` compnent to toggle RHDnsService.

2/ Define custom `SingleNodeCheckbox` component which will be very similar to `Checkbox` component but it will have custom `isChecked` and `onChange` use field's `setValue` formik helper to set it's value.

Second option is IMHO better as it requires only one new form value.

Reply:
Without having the value tracked by formik form's 'touched' and other helpers won't work as they won't take the single node value in account.